couchdbkit/version.py

```diff
@@ -4,5 +4,5 @@
 # See the NOTICE for more information.
 
 from __future__ import absolute_import
-version_info = (0, 9, 2)
+version_info = (0, 9, 3)
```

Comment:
why not use semver? @nickpell

Reply:
Hi @kkamalov, thanks for the feedback.  This fork of couchdbkit is intended for internal Dimagi use only, so we don't attempt to rigorously follow semantic versioning.  If you know of any other projects that use this fork, please let us know! AC